src/app/mesh/qgsmeshrendereractivedatasetwidget.cpp

```diff
@@ -36,11 +36,21 @@ QgsMeshRendererActiveDatasetWidget::QgsMeshRendererActiveDatasetWidget( QWidget
   connect( mPreviousDatasetButton, &QToolButton::clicked, this, &QgsMeshRendererActiveDatasetWidget::onPreviousTimeClicked );
   connect( mNextDatasetButton, &QToolButton::clicked, this, &QgsMeshRendererActiveDatasetWidget::onNextTimeClicked );
   connect( mLastDatasetButton, &QToolButton::clicked, this, &QgsMeshRendererActiveDatasetWidget::onLastTimeClicked );
+  connect( mDatasetPlaybackButton, &QToolButton::clicked, this, &QgsMeshRendererActiveDatasetWidget::onDatasetPlaybackClicked );
+
 
   connect( mDatasetGroupTreeView, &QgsMeshDatasetGroupTreeView::activeScalarGroupChanged,
            this, &QgsMeshRendererActiveDatasetWidget::onActiveScalarGroupChanged );
   connect( mDatasetGroupTreeView, &QgsMeshDatasetGroupTreeView::activeVectorGroupChanged,
            this, &QgsMeshRendererActiveDatasetWidget::onActiveVectorGroupChanged );
+
+  mDatasetPlaybackTimer = new QTimer( this );
+  connect( mDatasetPlaybackTimer, SIGNAL( timeout() ), this, SLOT( datasetPlaybackTick() ) );
+  mDatasetPlaybackButton->setIcon( QIcon::fromTheme( "media-playback-start" ) );
```

Comment:
This should use QgsApplication::getThemeIcon instead

Reply:
Thanks for the review.
However this would require me to change the function itself because icons I am using are not in the qgis/images/themes/
I can either
1] keep it as it is
2] modify QgsApplication::getThemeIcon so it falls back into QIcon::fromTheme
3] add Qt icons into the qgis/images/themes/

Reply:
I assume that 3 is the best option, this would also allow the icons to be reused with other temporal files.

Reply:
@roya0045 I am not sure if it is OK license-wise to copy icons from QT/System to qgis, do we have some precedence?

Reply:
@PeterPetrik ah I thought they were custom icons, like our others. :/